sensu-plugins-network-checks.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'sensu-plugin',  '~> 1.2'
   s.add_runtime_dependency 'net-ping',      '1.7.8'
   s.add_runtime_dependency 'whois',         '>= 4.0'
-  s.add_runtime_dependency 'whois-parser',  '1.0.0'
+  s.add_runtime_dependency 'whois-parser',  '1.0.1'
```

Comment:
rather than hard locking it lets pin it to the minor:
```
s.add_runtime_dependency 'whois-parser',  '~> 1.0.0'
```
Which should pull in any new patches but not new features.